src/renderer/search-results-component.ts

```diff
@@ -43,13 +43,8 @@ export const searchResultsComponent = Vue.extend({
             }
         },
         update(searchResults: SearchResultItem[]) {
-            let counter = 0;
-
             const viewModel = searchResults.map(
-                (searchResult): SearchResultItemViewModel => {
-                    counter++;
-                    return createSearchResultItemViewModel(searchResult, counter);
-                },
+                (searchResult, counter): SearchResultItemViewModel => createSearchResultItemViewModel(searchResult, counter),
```

Comment:
In your implementation counter is `0` for the first item and before the count started at `1`.

Reply:
Here is what I would suggest:

```typescript
const viewModel = searchResults.map(
    (searchResult, index): SearchResultItemViewModel =>
        createSearchResultItemViewModel(searchResult, index + 1),
);
```

Reply:
Whoops, fixed :)